packages/compute-baseline/src/browser-compat-data/feature.ts

```diff
@@ -72,34 +75,65 @@ export class Feature {
     return this.data.__compat?.status?.standard_track ?? false;
   }
 
-  _supportedBy(
-    browser: Browser,
-  ): { release: Release; qualifications?: Qualifications }[] {
+  rawSupportStatements(browser: Browser): SimpleSupportStatement[] {
     const support = this.data?.__compat?.support;
     if (support === undefined) {
-      throw Error("This feature contains no __compat object.");
+      throw new Error("This feature contains no __compat object.");
     }
 
     const statementOrStatements = support[browser.id];
-
     if (statementOrStatements === undefined) {
-      throw Error(`${this} contains no support data for ${browser.name}`);
+      throw new Error(`${this} contains no support data for ${browser.name}`);
     }
 
-    const rawStatements = Array.isArray(statementOrStatements)
+    return Array.isArray(statementOrStatements)
       ? statementOrStatements
       : [statementOrStatements];
+  }
+
+  supportStatements(browser: Browser): SupportStatement[] {
+    return this.rawSupportStatements(browser).map((raw) =>
+      statement(raw, browser, this),
+    );
+  }
 
+  supportedIn(release: Release): (Supported | Unsupported | UnknownSupport)[] {
     const result = [];
-    for (const raw of rawStatements) {
-      const s = statement(raw, browser, this);
+    for (const s of this.supportStatements(release.browser)) {
+      this.assertRealSupportStatement(s, release.browser);
 
-      if (!(s instanceof RealSupportStatement)) {
-        throw Error(
-          `${this.id} contains non-real values for ${browser.name}. Cannot expand support.`,
-        );
```

Comment:
This also happens in a couple of places, so I consolidated that too—it's at the end. I have no idea how to order things. 😬

Reply:
Order is hard. End is fine.